index.html

```diff
@@ -434,19 +434,19 @@ <h3>Attributes and default actions</h3>
                                     <td>Yes</td>
                                     <td>Yes</td>
                                     <td>Varies: when the pointer is primary, all default actions of the <code>mousedown</code> event
-                                        <br>Canceling this event also sets the <code>PREVENT MOUSE EVENT</code> flag for this <code>pointerType</code>, which prevents subsequent firing of certain <a>compatibility mouse events</a>.</td>
+                                        <br>Canceling this event also sets the <code>PREVENT MOUSE EVENT</code> flag for this <code>pointerType</code>, which prevents subsequent firing of <a>compatibility mouse events</a>.</td>
```

Comment:
When I just took a fresh look at this, it seems to me that `PREVENT MOUSE EVENT` flag is just an algorithmic details that is fully defined in Sec11.  I am proposing to remove it from here and also from the note in 4.2.3.

So this para would look like this:
> Canceling this event also prevents subsequent firing of compatibility mouse events.

Wdyt?